submit-queue/pod.yaml

```diff
@@ -3,9 +3,12 @@ kind: Pod
 metadata:
   name: submit-queue
   namespace: default
+  labels:
+    app: submit-queue
+    version: "0.4"
```

Comment:
I'll have to think about how to generate this in the makefile, where we know the Tag